web/models/user.ex

```diff
@@ -7,6 +7,13 @@ defmodule CodeCorps.User do
     field :password, :string, virtual: true
     field :encrypted_password, :string
 
+    field :first_name, :string
+    field :last_name, :string
+
+    field :twitter, :string
+    field :biography, :string
+    field :website, :string
+
```

Comment:
To me, it's easier to grasp which fields we have when they're grouped together in some sensible way. Alphabetising makes sense when there are too many fields to do any other type of ordering.